Engines/Wine/Verbs/dotnet45/script.js

```diff
@@ -42,8 +42,11 @@ Wine.prototype.dotnet45 = function() {
     this.overrideDLL()
         .set("native", ["mscoree"])
         .do();
-        
-    this.windowsVersion("win2003");
-
+    
```

Comment:
Can we replace this by a warning?
Something like:
```Javascript
if (this.windowsVersion != "win2003") {
    print(tr("Windows version must be \"win2003\" such that {0} applications can work."), ".NET 4.5");
}
```

(untested)

Reply:
As the windows version is set to win7 at the beginning of the script, there is no need of the "if".

Reply:
Ok, sorry I didn't see that. In any case, this has the same problem as setting "win2003" in the verb: it's not clear from the outside that the Windows version is changed. Apparently, "win7" is needed for the installation. I would therefore suggest the following:
1. store the Windows version
2. set to win7
3. install .Net
4. restore the Windows version
5. if restores Windows version != win2003, print warning

Reply:
{0} isn't replaced by .NET 4.5 for an obscure reason

Reply:
Should `.NET 4.5` be passed to `tr` or to `print`?

Reply:
Yes to `tr`. I hope that works in JS.

@ImperatorS79 can you try
```Javascript
if (this.windowsVersion != "win2003") {
    print(tr("Windows version must be \"win2003\" such that {0} applications can work.", ".NET 4.5"));
}
```

Reply:
Everything seems to work now